app/javascript/tests/util.js

```diff
@@ -0,0 +1,35 @@
+import moment from 'moment';
+
+/** formats patient name as it is done in Sara */
```

Comment:
Really small thing - could you update these comments to not say "as it is done in Sara" and just say "...as done when field is displayed." or something along those lines for clarity.

Reply:
all set